pysatModels/models/pydineof_dineof.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+"""
+Supports exported model data from pyDINEOF, a Python package
+that interfaces with a version of Data Interpolation Empirical
+Orthogonal Functions (DINEOFs). This module couples into
+the systematic export pyDINEOF format and thus should support
+all exports from the package.
+
+Given the generality of the support, each model series is identified
+using the `tag` keyword.
+
+DINEOFs are a purely data based method that can analyze a data-set, with
+data gaps, and extract a series of basis functions that optimally reproduce
+the input data. The quality of the reconstruction is primarily determined
+by the quantity and quality of the input data.
+
+http://modb.oce.ulg.ac.be/mediawiki/index.php/DINEOF
+
+References
+----------
+J.-M. Beckers and M. Rixen. EOF calculations and data filling from
+incomplete oceanographic data sets. Journal of Atmospheric and
+Oceanic Technology, 20(12):1839-­1856, 2003.
+
+Properties
+----------
+platform : string
+    pysat
+name : string
+    dineof
+inst_id : string
+    ['']
+tag : string
+    [*]
+
+Note
+----
+Specific tags are not listed here as this method is intended to support
+all pyDINEOF export models. Place the desired model (daily files)
+at '{pysat_data_dir}/pydineof/dineof/{tag}'. It is presumed the default
+naming scheme of 'dineof_{year:04d}-{month:02d}-{day:02d}.nc'
+has been retained. Use the file_format option for custom filenames.
+::
+    imodule = pysatModels.instruments.pydineof_dineof
+    model = pysat.Instrument(inst_module=imodule, tag=tag)
+
+"""
+
+import logging
+
+import pysat
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Look up the new instrument template to re-organize and add block comment headers.